integration/querydsl/expressions/src/main/java/com/blazebit/persistence/querydsl/JPQLNextTemplates.java

```diff
@@ -147,7 +147,9 @@ public JPQLNextTemplates(char escape, QueryHandler queryHandler) {
 
 
         add(JPQLNextOps.LEAST, "LEAST({0}, {1})");
+        add(Ops.MathOps.MIN, "LEAST({0}, {1})");
```

Comment:
`MathOps.MIN` is `MIN(a,b)` which in Blaze-Persistence is `LEAST`. This makes sure that QueryDSL's `MathExpressions.max(a, b)` uses the right dialect specific translation.

However, we still need `JPQLNextOps.LEAST`, because the aforementioned method only accepts numbers where `LEAST` can actually compare any comparable basic value.